docs/e2e-tests.md

```diff
@@ -64,19 +64,18 @@
 
 4. Deploy a cluster and run smoke test
 
-    Note that 'k8s-azure e2e' command will delete the cluster as last running step. To skip that, add parameter '-cskipcleanup=1' or set environment variable 'K8S_AZURE_SKIPCLEANUP=1'.
-
+    To deploy a cluster:
     ```
-    source <TestProfile>
-    CLUSTER_NAME=<ClusterName>
-    tests/k8s-azure/k8s-azure e2e -cname=$CLUSTER_NAME -cbuild_e2e_test=1 -caccm_image=<username>/azure-cloud-controller-manager:<image_version>
+    make deploy
```

Comment:
some env variables are required before deployment: https://github.com/kubernetes/cloud-provider-azure/blob/master/hack/deploy-cluster.sh#L26

IMAGE_REGISTRY should also be set before running make deploy.

Reply:
Sir , it is already there in Step 3.
3. Build a custom image and push it to a testing repository.
    ```
    IMAGE_REGISTRY=<username> make image
    docker push <username>/azure-cloud-controller-manager:<image_version>
    ```

Reply:
yep, but the variable name is different now. We should update them according to latest deploy-cluster.sh

Reply:
Okay Sir